releases/release-5.3.0.md

```diff
@@ -0,0 +1,395 @@
+| [`tidb_enable_pseudo_for_outdated_stats`](/system-variables.md#tidb_enable_pseudo_for_outdated_stats-new-in-v530) | Newly added | Controls the behavior of the optimizer when the statistics on a table expire. The default value is `ON`. When the number of modified rows in the table is greater than 80% of the total rows (This ratio can be adjusted by the configuration [`pseudo-estimate-ratio`](/tidb-configuration-file.md#pseudo-estimate-ratio)), the optimizer considers that the statistics other than the total number of rows are no longer reliable and use pseudo statistics instead. When you set the value as `OFF`, even if the statistics expire, the optimizer still uses them. |
+|[`tidb_enable_tso_follower_proxy`](/system-variables.md#tidb_enable_tso_follower_proxy-new-in-v53) | Newly added  | Determines whether to enable or disable the TSO Follower Proxy feature. The default value is `OFF`, which means the TSO Follower Proxy feature is disabled. At this time, TiDB only gets TSO from PD leader. When this feature is enabled, TiDB evenly sends the requests to all PD nodes when acquiring TSO. The PD follower then forwards the TSO requests to reduce the CPU pressure of PD leader. |
+|[`tidb_tso_client_batch_max_wait_time`](/system-variables.md#tidb_tso_client_batch_max_wait_time-new-in-v53) | Newly added |  Sets the maximum waiting time for a batch saving operation when TiDB requests TSO from PD. The default value is `0`, which means no additional waiting. |
+| [tidb_tmp_table_max_size](/system-variables.md#tidb_tmp_table_max_size-new-in-v53) | Newly added  | Limits the maximum size of a single [temporary table](/temporary-tables.md). If the temporary table exceeds this size, an error will occur. |
+
+### Configuration file parameters
+
+|  Configuration file    |  Configuration item  | Change type |  Description  |
+| :---------- | :----------- | :----------- | :----------- |
+| TiDB | [`prepared-plan-cache.capacity`](/tidb-configuration-file.md#capacity)  | Modified | Controls the number of cached statements. The default value is changed from `100` to `1000`.|
+| TiKV | [`storage.reserve-space`](/tikv-configuration-file.md#reserve-space) | Modified | Controls space reserved for disk protection when TiKV is started. Starting from v5.3.0, 80% of the reserved space is used as the extra disk space required for operations and maintenance when the disk space is insufficient, and the other 20% is used to store temporary files. |
+| TiKV | memory-usage-limit | Modified  | This configuration item is new in TiDB v5.3.0 and its value is calculated based on storage.block-cache.capacity. |
+| TiKV | `raftstore.raft-reject-transfer-leader-duration`  | Deleted | Determines the smallest duration that a Leader is transferred to a newly added node. |
```

Comment:
```suggestion
|  TiKV | `raftstore.store-pool-size` | Newly added | The allowable number of threads in the pool that handles Raft. |
|  TiKV | `raftstore.raft-write-size-limit` | Newly added | The threshold value that triggers Raft to write data. If the data size is larger than the value of this configuration item, the data is written to the disk. When the value of `raftstore.store-io-pool-size` is `0`, this configuration item does not take effect.|
|  TiKV | `raftstore.raft-msg-flush-interval` | Newly added | The interval between sending Raft messages in batch. The batch Raft messages are sent every interval specified by this configuration item. When the value of `raftstore.store-io-pool-size` is `0`, this configuration item does not take effect. |
| TiKV | `raftstore.raft-reject-transfer-leader-duration`  | Deleted | Determines the smallest duration that a Leader is transferred to a newly added node. |
```

Reply:
@TomShawn @gengliqi These are the newly added Raft Async I/O related configuration items, PTAL, thanks!

Reply:
raftstore.raft-msg-flush-interval: Determines the interval at which Raft messages are sent in batches. The Raft messages in batches are sent every interval specified by this configuration item.

Reply:
raftstore.raft-write-size-limit: Determines the threshold at which Raft data is written into the disk.